samples/ConsoleApp.SelfBootstrap/Program.cs

```diff
@@ -28,7 +44,7 @@ private static async Task<int> Main()
             });
 
         _tracerProvider = builder.Build();
-            
```

Comment:
Q.: don't we have a StyleCop to enforce no whitespace lines? Or it wasn't being enforced for this file?

Reply:
It was not enforced for sample folder. Now all projects should be covered by Stylecop.Analyzers.